hadoop-ozone/dist/src/main/smoketest/admincli/testdiskbalancer.robot

```diff
@@ -39,3 +45,13 @@ Check success with non-admin user for status and report diskbalancer
                         Should Contain                  ${result}                Status result:
     ${result} =         Execute                         ozone admin datanode diskbalancer report
                         Should Contain                  ${result}                Report result:
+
+Check if balancer stops automatically
+    Run Keyword         Kinit test user                 testuser                testuser.keytab
+    Execute             ozone admin datanode diskbalancer start -a
+
+    # Wait until the balancer status contains "RUNNING", retry every 5s for up to 1 min
+    Wait Until Keyword Succeeds   1 min   5 sec   Check Balancer Status   RUNNING
+
+    # Wait until the balancer status contains "STOPPED", retry every 5s for up to 2 min
+    Wait Until Keyword Succeeds   2 min   5 sec   Check Balancer Status   STOPPED
```

Comment:
@Gargi-jais11 , please set the "service.interval" value of disk balancer configuration to a small value in docker-config file for every compose directory which will run this testdiskbalancer.robot, so we can wait a smaller amount of time than 1m and 2m.